src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2148,20 +2148,23 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     info[jss::validation_quorum] = static_cast<Json::UInt>(
         app_.validators ().quorum ());
 
-    if (auto when = app_.validators().expires())
-    {
-        if(human)
-            info[jss::validator_list_expires] = to_string(*when);
-        else
-            info[jss::validator_list_expires] =
-                static_cast<Json::UInt>(when->time_since_epoch().count());
-    }
-    else
+    if (admin)
     {
-        if(human)
-            info[jss::validator_list_expires] = "unknown";
+        if (auto when = app_.validators().expires())
+        {
+            if (human)
+                info[jss::validator_list_expires] = to_string(*when);
```

Comment:
What do you think about displaying something like `"never"` or `"no expiration"` if `when` equals  `TimeKeeper::time_point::max()`? Same would apply to the `validator_lists` response

Reply:
I like "never". I'll update it.